configure

```diff
@@ -3514,6 +3514,28 @@ if test -n "$_host"; then
 			_backend="opendingux"
 			_port_mk="backends/platform/sdl/opendingux/opendingux.mk"
 			add_line_to_config_mk 'OPENDINGUX = 1'
+			add_line_to_config_mk "OPENDINGUX_TARGET = ${_host#opendingux-}"
+			case "$_host" in
+				opendingux-gcw0)
+					_opengl_mode=gles2
+					_opengl_game=yes
```

Comment:
does it have desktop opnegl?

Reply:
Nope, gles1/2 and gl4es

Reply:
gl4es acts for desktop opengl. just making sure it was not mistake